src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -1702,6 +1702,28 @@ int p
                   } = 1;
                 }
                 """);
+            TestNormalizeDeclaration(
+                "class A{public string Prop{get;}public void Foo(){}}", """
+                class A
+                {
+                  public string Prop { get; }
+
+                  public void Foo()
+                  {
+                  }
```

Comment:
What if there wasn't a method here?

Reply:
100% suspected this question) There are already test cases for this:
https://github.com/dotnet/roslyn/blob/683e22d4a2351e5495e18fbb2f7089f2f1a9679d/src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs#L1159-L1184
It just very luckely appeared that when the next token is a `}`, execution falls to other branch, where it determines only 1 line break.